### DISCUSSION TITLE
Restrict invites by top level domain

### DISCUSSION THREAD

**Discussion by ackernaut** [ParabolInc/parabol#5663] [Feedback]
[This customer](https://app.hubspot.com/contacts/3888472/contact/40608001/) 🔒  was surprised to find out that users within their account could invite anybody (any email address), and that those users were also able to do the same. In Loom, [at some point after 27:14](https://www.loom.com/share/242b2fab85da4d0e90055e253c0c4a91) 🔒 , he suggested the idea of being able to restrict this by email domain.

I’m opening this discussion to gather any similar feedback from other customers, and to see if this is something we should explore. Is this a feature? Is it a paid feature?

**Related**: there is an issue about designing functionality to restrict invites and new sign-ups to SSO/SAML users, see #5512 — is this how we solve for this?

**Comment by gcrickman** [comment]
This Enterprise [customer](https://app.hubspot.com/contacts/3888472/contact/26318951?interaction=email) 🔒  also requested:

> I would like to know how can we restrict access to certain domain for our organization? We would like to just have @xxxx.com email addresses.
> Also, how can we control new invites? Only org admins can send invite to new people.

**Comment by gcrickman** [comment]
This [customer](https://app.hubspot.com/contacts/3888472/deal/5051420932) 🔒  requires restrictions in order to use their private managed instance. Currently, we have implemented IP restrictions to try to solve this but a domain lock would solve this problem.

**Comment by avivapinchas** [comment]
This is now available as an Enterprise feature!

**Reply by ackernaut** [reply]
Woo hoo! Are we getting back with folks who asked for this? At any rate I imagine it’ll come up with any forward motion with customers
